DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/src/simple.cpp

```diff
@@ -0,0 +1,84 @@
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <iostream>
+
+using namespace sycl;
+constexpr access::mode sycl_read = access::mode::read;
+constexpr access::mode sycl_write = access::mode::write;
+
+class SimpleAdd;
+class SimpleMul;
+
+void kernelAdd(const float a, const float b, float &c) {
```

Comment:
Function names should be CamelCase
```suggestion
void KernelAdd(const float a, const float b, float &c) {
```